README.md

```diff
@@ -193,6 +194,8 @@ This option is mainly meant to be toggled with the `Toggle auto Reset Image` hot
 - **Custom thresholds** are place between parenthesis `()` in the filename. This value will override the default threshold.
 - **Custom pause times** are placed between square brackets `[]` in the filename. This value will override the default pause time.
 - **Custom delay times** are placed between hash signs `##` in the filename. Note that these are in milliseconds. For example, a 10 second split delay would be `#10000#`. You cannot skip or undo splits during split delays.
+- **Custom comparison FPS** is placed between exclamation marks `!!` in the filename. This value will override the default FPS for the given image or text file and does also apply to the reset image.
+- **Custom rectangle position** is placed between plus signs `++` in the filename. Note that these will only apply for text files when using text recognition (.txt). The scheme looks like this: `+740-1180-60-150+`. These are the X and Y coordinates in the image to draw a rectangle. They are seperated by the minus sign `-` and follow the form `+X-X-Y-Y+`. The second X and Y values need to be bigger then the first ones. You will need to adjust these values depending on your capture resolution.
```

Comment:
I'll add the comment here, but it applies for all relevant code changes.
I'd *really* rather not add more file-specific settings unless necessary.

1. We're very limited on usable, meaningful and easy to use symbols
2. I'd like to move away from this system in v3
3. This could break someone's existing setup if they use either symbols

Moreso if it's specific to a comparison method

---
**Custom Rectangle position**

I'd rather we encode the rectangle information as part of the file data. For example, a simple TOML-parseable file could be enough.
The advantage of using a key-value serialization is that it's more human readable, easier to reason about for regular users, easier to give relevant errors and help if the file is invalid AND is more future-proof if we need to add further details.

For example (key names tbd, if you have a better idea):
```toml
data = "The text to find"
top_left = 0
top_right = 0
bottom_left = 0
bottom_right = 0
```
vs
```txt
The text to find
0
0
0
0
```
or 
```txt
The text to find;0;0;0;0
```

If you don't see this being expanded in the future, and think is overkill, I can accept using `+X-X-X-X+` for OCR

---

About FPS: I'd be fine adding a global "text recognition" FPS setting instead. The code in that area is cleaner/safer (I've already refactored it to be a mostly type-safe `TypeDict` that should warn about any place you need to update the code) and it's more user-friendly if they don't need to encode it into every text file.

Reply:
Noted. I'll use a config file for all the settings.